spec/cohort-manager.spec.js

```diff
@@ -0,0 +1,48 @@
+import { CohortManager } from '../src/cohort-manager.js'
+import { Cohort } from '../src/cohort.js'
+
+describe('Manager', () => {
+  let manager
+  beforeEach(() => {
+    manager = new CohortManager()
+  })
+  it('updates the list property', () => {
+    const cohort = new Cohort('purple', manager)
+    const result = manager.setList(cohort)
+    expect(result[0].id).toEqual(undefined)
+    expect(result[0].cohortName).toEqual('purple')
+    expect(result[0].students).toEqual([])
```

Comment:
A little tip here for a minor readability improvement - you can destructure arrays in the same way you do with objects.

```js
const [createdCohort] = manager.setList(cohort)
expect(createdCohort.id).toEqual(undefined)
...
```